### ISSUE TITLE
Trying to export image to PDF,image OK, searchable text NO 

### ISSUE THREAD

**Issue by ericduarte** [r1me/TTesseractOCR4#3]
Hello

I've tried to export image to PDF, it generates the pdf, but the text is not searchable.

**Comment by r1me** [comment]
I assume you've downloaded and provided path to language data and specified it in call to `Tesseract.Initialize`, default is English. If trained data doesn't match language that you want to OCR, PDF file will be created but there will be no text to search in it.

Please try OCR your image with `examples\delphi-console-simple` example, and post your results (is text returned in console).

**Comment by ericduarte** [comment]
Thanks for your attention,

this examples\delphi-console-simple, works fine
but examples\delphi-console-pdfconvert, no

`program delphi_console_pdfconvert;

{$APPTYPE CONSOLE}

{$R *.res}

uses
  tesseractocr in '..\..\tesseractocr.pas';

var
  inputFileName,
  outputFileName: String;
begin
  Tesseract := TTesseractOCR4.Create;
  try
    if Tesseract.Initialize('tessdata\', 'eng') then
    begin
      inputFileName := 'samples\multi-page.tif';
      outputFileName := 'multi-page.pdf';

      if Tesseract.CreatePDF(inputFileName, outputFileName) then
      begin
        WriteLn('PDF was saved succesfully to ' + outputFileName);
        ReadLn;
      end;
    end;
  finally
    Tesseract.Free;
  end;
end.`

**Comment by r1me** [comment]
Please attach input image and output PDF.

**Comment by ericduarte** [comment]
I compressed the image to attach

[multi-page.zip](https://github.com/r1me/TTesseractOCR4/files/1470822/multi-page.zip)
[multi-page.pdf](https://github.com/r1me/TTesseractOCR4/files/1470816/multi-page.pdf)

**Comment by ericduarte** [comment]
I'm using Delphi 10 Seatle, and did a litle change in tesseractocr.consts.pas,
 included it 

`{$IFDEF VER300}
type
  PUTF8Char = PAnsiChar;
{$ENDIF}`

 and changed it

`{$IFDEF Use_CPPAN_Binaries}
  libleptonica = {$IFDEF Linux}'libpvt.cppan.demo.danbloomberg.leptonica-1.74.4.so'{$ELSE}'liblept-5.dll'{$ENDIF};
  libtesseract = {$IFDEF Linux}'libpvt.cppan.demo.google.tesseract.libtesseract-master.so'{$ELSE}'libtesseract-4.dll'{$ENDIF};
  {$ELSE}`

**Comment by r1me** [comment]
I compared `multi-page.pdf` that I'm getting with yours, and can say with no doubt the issue is in Tesseract. OCR export to PDF is still under development of Tesseract, latest branch will even crash while trying to save to PDF file. I've made a copy of [build dated 07-08-2017, this one](http://r1me.pl/tesseractocr-master-0708.zip) seems to create searchable PDF files:

![search_pdf](https://user-images.githubusercontent.com/13695767/32781771-824033ac-c946-11e7-9c96-113efb774900.png)

Thanks for finding this issue. I will monitor Tesseract development and update precompiled binaries on my server, once issue is fixed in the Tesseract.
